index.html

```diff
@@ -0,0 +1,134 @@
+                        <p class="text--unit">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim 
+                            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip 
+                            ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+                            velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat 
+                            cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </p>
+                    </section>
+                    <section class="last-info__unit">
+                        <p class="title--unit">Заголовок</p>
+                        <p class="text--unit">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim 
+                            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip 
+                            ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+                            velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat 
+                            cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </p>
+                    </section>
+                </section>
+            </section>
+            <section class="interesting-article">
+                <header class="title--double">
+                    <p class="title--double__heading">Интересные статьи</p>
+                    <p class="title--double__supporting-text">Последние новости из мира IT-технологий</p>
+                </header>
+                <div class="posts">
+                    <div class="blog">
+                        <div class="blog__img">
+                            <img class="blog-img" src="img/article1.jpg" alt="">
+                        </div>
+                        <div class="blog__content">
+                            <div class="blog__content--title">Зачем нужны страница или экран успеха</div>
+                            <div class="blog__content--info">
+                                <p>
+                                    После того как пользователь совершил целевое действие, например, 
+                                    оформил заказ, заполнил форму и нажал кнопку «Отправить», подтвердил адрес электронной почты 
+                                    и т.д., открывается страница или экран успеха (благодарности). Зачем они нужны? Давайте разберемся.
+                                </p>
+                            </div>
+                            <div class="blog__content--date">17 окт 2022 • 11 мин чтения</div>
+                        </div>
+                    </div>
```

Comment:
все можно сделать проще, один блог, это компонент

blog__content--title, blog__content--info это модификаторы, модификаторы применяются так

blog__title blog__title--primary и.тд, в лекциях же все написано

```suggestion
    <div class="blog">
        <div class="blog__media">
            <img class="blog__img" src="img/article1.jpg" alt="">
        </div>
        <div class="blog__content">
            <div class="blog__title">Зачем нужны страница или экран успеха</div>
            <div class="blog__info">
                <p>
                    После того как пользователь совершил целевое действие, например, 
                    оформил заказ, заполнил форму и нажал кнопку «Отправить», подтвердил адрес электронной почты 
                    и т.д., открывается страница или экран успеха (благодарности). Зачем они нужны? Давайте разберемся.
                </p>
            </div>
            <div class="blog__date">17 окт 2022 • 11 мин чтения</div>
        </div>
    </div>
```